Dockerfile

```diff
@@ -34,7 +34,7 @@ ENTRYPOINT ["./tools/dev/telemetry_mock_api.sh"]
 # This image gets grpc health check probe
 FROM build_base AS grpc_health_probe_builder
 ARG TARGETARCH
-RUN GRPC_HEALTH_PROBE_VERSION=v0.4.22 && \
+RUN GRPC_HEALTH_PROBE_VERSION=v0.4.27 && \
```

Comment:
the latest version is `v0.4.29` can we use it?

Reply:
We can't, as the version isn't available in the release section yet: https://github.com/grpc-ecosystem/grpc-health-probe/releases
When passing that version, v0.4.29, it will fail on retrieving it, as it isn't available yet.